front/src/modules/activities/components/ActivityEditor.tsx

```diff
@@ -185,7 +185,7 @@ export function ActivityEditor({
               <>
                 <DateEditableField
                   value={activity.dueAt}
-                  icon={<IconCalendar />}
+                  Icon={IconCalendar}
```

Comment:
we also need to put this in the doc: when a props is a component, we must write it in PascalCase